src/v2/api/teacher/DisciplineTeacherService.ts

```diff
@@ -278,4 +278,21 @@ export class DisciplineTeacherService {
     }
     await this.disciplineTeacherRepository.deleteById(disciplineTeacher.id);
   }
+
+  async getManyByTeacherId (teacherId: string, userId: string) {
```

Comment:
The name of method is misunderstanding, because you haven't just find many (?) by teacher id, but also user id.

It's better to show the sense of method, for example `getByTeacherAndUser` or `getUserDisciplineTeachers`, etc.